### PR TITLE
user/cmatrix: new package

### PR DESCRIPTION
**Pull request by NovaAndrom3da**

### CODE REVIEW

File: user/cmatrix/template.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+_commit = "5c082c64a1296859a11bee60c8c086655953a416"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja"]
+makedepends = ["ncurses-devel"]
+pkgdesc = "'The Matrix'-like effect for terminals"
+maintainer = "Nova <froggo8311@proton.me>"
+license = "GPL-3.0-only"
+url = "https://github.com/abishekvashok/cmatrix"
+source = f"{url}/archive/{_commit}.zip"
```
Comment (nekopsykose) [inline user/cmatrix/template.py:12]:
```suggestion
source = f"{url}/archive/{_commit}.tar.gz"
```

File: user/cmatrix/template.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+pkgname = "cmatrix"
+pkgver = "2.0"
```
Comment (nekopsykose) [inline user/cmatrix/template.py:2]:
```suggestion
pkgver = "2.0_git20231209"
```

**Comment by nekopsykose** [comment]
-c still seems to not work which is weird cause it did before

**Comment by q66** [comment]
no changes in ages, git package, etc i'll close for now

**Pull request closed without merging**